html/global_attributes.json

```
@@ -1016,9 +1016,26 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": false
-            },
+            "firefox": [
+              {
+                "version_added": "23",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.forms.inputmode",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": [
+                  "Prior to version 77, Firefox did not support <code>inputmode</code> when <code>contenteditable</code> is <code>true</code>.",
```

@Elchi3, since you're taking ownership of this, can I suggest some changes to the notes (I've just been thinking about our style because #6229)? These are optional, if you just want to merge without.

```suggestion
                  "Before version 77, Firefox does not support <code>inputmode</code> when <code>contenteditable</code> is <code>true</code>.",
```